### PR TITLE
dg box approval section

### LINKED ISSUE #11: Admin Dashboard: Box Approval/Denied Section
- Develop frontend component for approval/denied section on the Admin Dashboard page
- Build a separate component for the individual Box Information with all the details listed on the mockup
- [Figma Sheet](https://www.figma.com/file/y30pb19CamcegHG7Oh4lJC/FYA-Low-Fidelity-Wireframes)

### PR DESCRIPTION
**Pull request by ctc-devops**
- Develop frontend component for approval/denied section on the Admin Dashboard page
- Build a separate component for the individual Box Information with all the details listed on the mockup
- [Figma Sheet](https://www.figma.com/file/y30pb19CamcegHG7Oh4lJC/FYA-Low-Fidelity-Wireframes)
closes #11

### CODE REVIEW

File: src/components/BoxApproval/BoxApproval.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <div className="boxApproval">
+      <h1 className="boxApprovalHeading">Box Approval</h1>
+      <div className="boxApprovalMenu">
+        <Stack spacing={0} direction="row" align="center" className="buttonStack">
+          <Button
+            onClick={() => setCurrentTab('requiresApproval')}
+            size="xs"
+            border="none"
+            width="100%"
```
Comment (meghakak) [inline src/components/BoxApproval/BoxApproval.jsx]: Currently, these two buttons show up as longer than the box approval component on my screen. I believe changing the width for both of these to 50% should fix this issue. Please try to resize your screen to make sure this (and other styling) looks good after this change.

File: src/components/BoxApproval/BoxApproval.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const [requiresApprovalBoxes, setRequiresApprovalBoxes] = useState([Box(), Box(), Box()]);
+  // eslint-disable-next-line no-unused-vars
+  const [approvedDeniedBoxes, setApprovedDeniedBoxes] = useState([Box(), Box()]);
+
+  return (
+    <div className="boxApproval">
+      <h1 className="boxApprovalHeading">Box Approval</h1>
+      <div className="boxApprovalMenu">
+        <Stack spacing={0} direction="row" align="center" className="buttonStack">
+          <Button
```
Comment (meghakak) [inline src/components/BoxApproval/BoxApproval.jsx]: In general, we want to separate our css styling and html/react components so let's move the styling on the buttons to the css file. Please do the same for Box.jsx.

File: src/components/BoxApproval/Box.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+function Box() {
+  return (
+    <div className="box">
+      <HStack className="boxHeader">
+        <VStack spacing={0} align="left" className="boxHeaderInfo">
+          <Text fontSize="6xl">Box #</Text>
+          <Text>date</Text>
+        </VStack>
+        <Spacer />
+        <Button size="xs" className="approveButton">
```
Comment (meghakak) [inline src/components/BoxApproval/Box.jsx]: Could we change the "No" and "Yes" to X and checkmark icons as shown in the Figma? Y'all can use the Chakra icons for now: https://chakra-ui.com/docs/media-and-icons/icon

File: src/components/BoxApproval/Box.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+          <p className="buttonText">No</p>
+        </Button>
+        <Button size="xs" className="approveButton">
+          <p className="buttonText">Yes</p>
+        </Button>
+      </HStack>
+      <VStack spacing={0} direction="column" align="left" className="boxContent">
+        <Text>ID</Text>
```
Comment (meghakak) [inline src/components/BoxApproval/Box.jsx]: The ID has an extra space below it. Setting margin-bottom to 0 should fix this!

**Review by meghakak — CHANGES_REQUESTED** [review]
Great work on this sprint! Left a couple comments for small things to fix. Let me know if you have any questions about them :)

**Comment by dgardezi** [comment]
I updated the code quite a bit to try and make things cleaner, but I believe all of the issues you addressed are fixed! Let me know if there is anything else to edit.

**Review by meghakak — APPROVED** [review]
LGTM! Thanks for making those changes, code looks much cleaner :)

**Review by sylvesteryue — APPROVED** [review]
LGTM as well!

**Pull request merged**